lib/src/androidTest/java/com/couchbase/lite/internal/core/C4BaseTest.java

```diff
@@ -138,24 +138,29 @@ private void deleteFile(String filename) {
 
     private byte[] createFleeceBody(Map<String, Object> body) throws LiteCoreException {
         FLEncoder enc = new FLEncoder();
-        enc.beginDict(body != null ? body.size() : 0);
-        for (String key : body.keySet()) {
-            enc.writeKey(key);
-            enc.writeValue(body.get(key));
-        }
-        enc.endDict();
-        return enc.finish();
+        try {
+            enc.beginDict(body != null ? body.size() : 0);
+            for (String key : body.keySet()) {
+                enc.writeKey(key);
+                enc.writeValue(body.get(key));
+            }
+            enc.endDict();
+            return enc.finish();
```

Comment:
`L123 => kFleeceBody = createFleeceBody(body);`

Do we need to free this kFleeceBody in the tearDown?

Reply:
No. It's java byte[] array. It will get GC.